modules/lib/lib-project/src/main/resources/lib/xp/project.ts

```diff
@@ -31,31 +31,31 @@ export interface ProjectReadAccess {
     public: boolean;
 }
 
-export interface CreateProjectParams {
+export interface CreateProjectParams<CONFIG extends Record<string, unknown> = Record<string, unknown>> {
```

Comment:
The same comments as above. Don't uppercase type parameters, and think through where we want default type parameters (I think we maybe only want them on functions).